src/app/public/modules/file-attachment/file-drop.component.ts

```diff
@@ -30,33 +30,57 @@ import {
   styleUrls: ['./file-drop.component.scss']
 })
 export class SkyFileDropComponent implements OnDestroy {
+  /**
+   * Fires when users add or remove files.
+   */
   @Output()
   public filesChanged = new EventEmitter<SkyFileDropChange>();
 
   @Output()
```

Comment:
Missing event description. Not sure if it's worth worrying about now given the push to migrate quickly.

Reply:
Done

Reply:
Sorry if I'm doing something wrong, but I still don't see this property descrption in the PR preview or in the content on the PR's Files tab.